templates/default/less/variables.less

```diff
@@ -90,16 +93,23 @@
 @border-radius-large:       0px;
 @border-radius-small:       0px;
 
+// This controls the heights of the header components.
```

Comment:
All variables must be placed in a section describing their functionality. Use the bootstrap conventUse to Bootstrap convention to describe sections (see Services/UICore/lib/bootstrap-3.2.0/less/variables.less): 

//== Section Name
//
//## Section Description

//** Description of variable
@variable:               value;